resource_obmcs_identity_ui_password_test.go

```diff
@@ -70,36 +55,6 @@ func (s *ResourceIdentityUIPasswordTestSuite) TestCreateUIPassword() {
 	})
 }
 
-func (s ResourceIdentityUIPasswordTestSuite) TestUpdateVersionForcesNewUIPassword() {
```

Comment:
Doesn't look like this is tested in the new version, but I'm also not convinced that it was being correctly tested in the old version either.

Reply:
I remember version being exposed at one point on passwords, but I thought it was swift passwords, and it's not present on a field of either in the api docs. So that particular test concept is obsolete. 

That said I think you are referring to a verification step that asserts that the new password object is different from the predecessor, something also difficult because neither password type has an ocid. 

Such a test would need to change the userId of the password, which is an invalid approach.